CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# CHANGELOG
+
+##  0.1.1
+* fixes bug where event publishing uses the wrong method name to look  up local events
+* fixes condition where events  may double publish when multiple events are registered
```

Comment:
double spaces

Reply:
not sure I follow

Reply:
events[space][space]may

Reply:
ah, there is a  bug in my stoopid keyboard doing that, I'll submit another PR with that fix in there